contrib/coreos/user-data.example

```diff
@@ -133,3 +134,14 @@ write_files:
       TOOLBOX_DOCKER_IMAGE=ubuntu-debootstrap
       TOOLBOX_DOCKER_TAG=14.04
       TOOLBOX_USER=root
+  - path: /etc/environment_proxy
+    owner: core
+    content: |
+      HTTP_PROXY=
+      HTTPS_PROXY=
+      ALL_PROXY=
+      NO_PROXY=
+      http_proxy=
+      https_proxy=
+      all_proxy=
+      no_proxy=
```

Comment:
@ianblenke builder runs custom buildpacks and we don't know what kind of program it will run, some programs use uppercase and others lowercase